.github/workflows/ci.yml

```diff
@@ -1,19 +1,38 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Can we not limit to `main` branch only? We have multiple active branches (main, develop, release/*, docs/*, alphanet, etc.) and personally I also use CI for feature branch.

Reply:
Cool. Let's include all the listed branches except feature branch.  I can create a draft PR for my feature branch to utilise the CI, if needed.

Reply:
Still think it's not right.  Both push and PR to any of the above branch should trigger a build.